addOns/bruteforce/src/main/java/com/sittinglittleduck/DirBuster/Manager.java

```diff
@@ -208,7 +208,7 @@ public Manager() {
     }
 
     // set up dictionay based attack with normal start
-    public void setupManager(
+    public synchronized void setupManager(
```

Comment:
Does it really need to be `synchronized`? The scan didn't start yet (the other `setupManager`s are not `synchronized`).